docs/howto/file_naming.rst

```diff
@@ -0,0 +1,16 @@
+File naming convention
+----------------------
+
+We recommend to use the following naming convention for migration files:
+
+.. code-block:: console
+
+    [VERSION]-[MIGRATION]-[ORDER]-[TYPE].sql
```

Comment:
An example would be great I think.

Reply:
Same here. All of the "how-to" content has been put there hastily, but we'll get a better look at it later.